src/main/java/com/google/api/codegen/util/NameFormatter.java

```diff
@@ -73,4 +73,9 @@
    *  Formats the base file name.
    */
   String classFileNameBase(Name name);
+
+  /**
+   * Formats the name into human readable form, useful in comments.
+   */
+  String humanName(Name name);
```

Comment:
This name feels odd. I might reasonably expect it to return "Michael". Maybe instead "phraseName()".